src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -50,6 +51,42 @@ export class RawFilesStackedTable extends React.PureComponent {
 
     }
 
+    /* Built-in headers for props.experimentSetType, extended by any matching title from props.columnHeaders */
```

Comment:
In this file, refactored an instance of componentWillReceiveProps into memoized functions and made a couple of comments/reminders to refactor remaining `componentWillReceiveProps` (now `UNSAFE_componentWillReceiveProps`) later.

No new features, all stacked file tables (e.g. Raw Files, Processed Files, for an ExpSet) should work/look exactly same as before.